openpype/modules/base.py

```diff
@@ -344,20 +311,29 @@ def _load_modules():
             ).format(dirpath))
             continue
 
+        is_in_current_dir = dirpath == current_dir
         for filename in os.listdir(dirpath):
             # Ignore filenames
             if filename in IGNORED_FILENAMES:
                 continue
 
+            if (
+                is_in_current_dir
+                and filename in IGNORED_DEFAULT_FILENAMES
+            ):
+                continue
+
             fullpath = os.path.join(dirpath, filename)
             basename, ext = os.path.splitext(filename)
 
+            # Validations
             if os.path.isdir(fullpath):
                 # Check existence of init file
                 init_path = os.path.join(fullpath, "__init__.py")
                 if not os.path.exists(init_path):
                     log.debug((
-                        "Module directory does not contain __init__.py file {}"
+                        "Module directory does not contan __init__.py"
```

Comment:
To preserve the typo fix from here: https://github.com/pypeclub/OpenPype/pull/3132
```suggestion
                        "Module directory does not contain __init__.py"
```